hasura/migrations/1645463938163_guarantee_players_exist_in_the_profile_table/up.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO profile (player_id)
```

Comment:
@dysbulic this should be closing #1114 ?

Reply:
✅